src/components/cookie-consent-drawer.js

```diff
@@ -0,0 +1,29 @@
+import { Button, Drawer } from 'antd';
+import React, { useState } from 'react';
+import cookie from '../common/cookie.js';
+import { useTranslation } from 'react-i18next';
+
+const EDUCANDU_COOKIE_CONSENT = 'educanduCookieConsent';
+
+export default function CookieConsentDrawer() {
+  const consentCookie = cookie.get(EDUCANDU_COOKIE_CONSENT);
+  const [isVisible, setIsVisible] = useState(!consentCookie);
+
+  const handleClose = () => {
+    cookie.set(EDUCANDU_COOKIE_CONSENT, 'true', { expires: 365 });
```

Comment:
I don't know when to make it expire, I feel like either on logout or when you close the browser tab (if you aren't logged in).

Reply:
I would get pretty mad if I would see it everytime I open the browser and go to the website, so I like the 365 :-)

Reply:
I've let it be